valtech-ch/saas-aem-module
# ICSAAS-362 search component integration test

- test the component's exported json model
 - add sample content package for the integration test
 - add expected json file
 - implement a sample content install rule
 - implement sling model json exporter client
